### PR TITLE
Tech/improve dx

### PR DESCRIPTION
**Pull request by laruiss**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 <html lang="fr">
 <head>
     <meta charset="UTF-8">
-    <base href="https://cristianpb.github.io/covid-19-certificate">
```
Comment (cristianpb) [inline index.html]: I think that if you delete this line, my github pages deploy will fail.

Comment (cristianpb) [reply index.html]: I think it's better to add this line at build time and being able to customize it

File: certificate.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
-const { PDFDocument, StandardFonts } = PDFLib
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+import { PDFDocument, StandardFonts } from 'pdf-lib'
+import QRCode from 'qrcode'
+
+import pdfBase from './certificate.pdf'
```
Comment (cristianpb) [inline certificate.js]: I think that certificate.pdf needs to be added to dist folder also, non ?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+$ npm start
+```
+
+### Générer le code de production
+
+```console
+$ npm run build
+```
+
+Le code à déployer sera dans le dossier `dist`
```
Comment (cristianpb) [inline README.md]: I would be changing this name folder to `docs` in order to continue having github pages deploy.

File: certificate.js (hunk 2 above)
Comment (laruiss) [reply certificate.js]: It is done on build. It has a hash on the name, as it should have (cache busting).

File: index.html (hunk 1 above)
Comment (laruiss) [reply index.html]: I don't see any reason why it should fail. I'll.try on my repo.

**Pull request merged**